components/MainDonationForm.js

```diff
@@ -19,11 +20,35 @@ const AMOUNTS = [
   { value: 25000, label: '$25,000' },
 ];
 
+const CAUSES = [
+  { id: 1, label: 'Students in Need' },
+  { id: 2, label: 'People of Color in Need' },
+  { id: 3, label: 'Immigrants in Need' },
+  { id: 4, label: 'Seniors in Need' },
```

Comment:
```suggestion
  { value: 4, label: 'Seniors in Need' },
```
The API for `SelectFormControl` looks for `value`, not `id`.

Reply:
The `CAUSES` is used by `CheckboxFormControl`, not by `SelectFormControl`. For checkboxes I just use `id` to keep information about which one is selected. Or am I missing something 🙂 ?